TESTING.md

```diff
@@ -5,64 +5,50 @@
 - [Integration tests](#integration-tests)
 - [Rake Tasks](#rake-tasks)
 
-## Syntax and style
+## Requirements
 
-### Requirements:
+- [ChefDK](https://downloads.chef.io/chefdk)
+- [Vagrant](https://www.vagrantup.com/)
+- [Packer](https://www.packer.io/)
+- A supported macOS hypervisor:
+  - [Parallels](https://www.parallels.com/landingpage/pd/general/)
+  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+  - [VMWare Fusion](https://www.vmware.com/products/fusion.html)
 
-- `cookstyle` and `foodcritic` (both are shipped with the [ChefDK](https://downloads.chef.io/chefdk))
```

Comment:
Is TESTING.md taken from Chef itself? I feel like it would be better to link to first party for `cookstyle` and `foodcritic`.

Reply:
No, @americanhanko wrote most of it IIRC.